src/ocamlorg_static/dune

```
@@ -25,4 +25,18 @@
     plain
     %{workspace_root}/data/media
     -o
-    media.ml))))
+    %{target}))))
+
+(rule
+ (target playground_digests.ml)
+ (deps
+  (source_tree %{workspace_root}/playground/asset))
+ (action
+  (with-stdout-to
+   %{null}
+   (run
+    %{bin:static-file-digest}
+    %{workspace_root}/playground/asset/
+    -s
+    -o
+    %{target}))))
```

I know this is ugly:

```suggestion
     %{target}
    (chdir %{workspace_root}/playground/asset
    (pipe-stdout
      (run %{bin:md5sum} %{deps})
      (run %{bin:awk} "{ print \"| \\\"\"$2\"\\\" -> Some (from_hex \\\"\"$1\"\\\")\"}")
      (system "(echo \"let digest = Digest.(function\"; cat -; echo \"| _ -> None)\")"))))))
```

But it turns an ad-hoc dependency into a system dependency.

What do you think?

---

I can see the appeal of having no dependency and this is really concise too, but if we do that we should do it as a separate script (with comments what it does).

---

You are right; it should be a script. Then if we have a script, it doesn't have to be a shell-script, it could be a OCaml-script.